utils/db/trigger.ts

```diff
@@ -0,0 +1,23 @@
+import conn from ".";
+
+export const getUserInfoFromDb = async (email: string) => {
```

Comment:
Why not directly use the `getUserByAlias` function from the `db/users.ts` instead of creating this new function?

Reply:
done